src/components/dashjs-player/dashjs-player.tsx

```diff
@@ -181,7 +286,7 @@ export class DashjsPlayer {
                 <div id="bitrateListBtn" class="control-icon-layout" title="Bitrate List">
                   <span class="icon-bitrate"></span>
                 </div>
-                <input type="range" id="volumebar" class="volumebar" min="0" max="1" step=".01" value="1" />
+                <input type="range" id="volumebar" class="volumebar" value="1" min="0" max="1" step=".01" />
```

Comment:
please return to the structure from before, this leads to warnings in the developer console

Reply:
reverted